boilerplate/openshift/golang-osd-operator/csv-generate/catalog-build.sh

```diff
@@ -31,18 +35,36 @@ if [ ! -d "${SAAS_OPERATOR_DIR}/.git" ] ; then
     exit 1
 fi
 
+# Calculate new operator version from bundles inside the saas directory
+OPERATOR_NEW_VERSION=$(ls "${BUNDLE_DIR}" | sort -t . -k 3 -g | tail -n 1)
+
+# Create package yaml
+# This must be included in the registry build
+# `currentCSV` must reference the latest bundle version included.
+# Any version their after `currentCSV` loaded by the initalizer
+# will be silently pruned as it's not reachable
+PACKAGE_YAML_PATH="${BUNDLE_DIR}/${operator_name}.package.yaml"
+
+cat <<EOF > "${PACKAGE_YAML_PATH}"
+packageName: ${operator_name}
+channels:
+- name: ${operator_channel}
+  currentCSV: ${operator_name}.v${OPERATOR_NEW_VERSION}
+EOF
+
+# Build registry
 cat <<EOF > $DOCKERFILE_REGISTRY
-FROM quay.io/openshift/origin-operator-registry:latest
+FROM quay.io/openshift/origin-operator-registry:4.7.0
```

Comment:
It would be nice if this `4.7.0` wasn't buried deep in this shell script; a good opportunity for some kind of configuration file.

Reply:
Agree, but for the moment at least we can make it consistent across all operators. We plan to hook this up to follow OCP releases.